recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: optv
+  version: 0.2.3
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/0.2.3.tar.gz
+
+build:
+  number: 0
+  script: cd py_bind && python setup.py prepare && python setup.py install
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - numpy x.x
+    - cython
+    - pyyaml
+
+test:
+  requires:
+    - python
+    - setuptools
+    - - {{ pin_compatible('numpy') }}
+    - scikit-image
+    - matplotlib
+    - scipy
+    - nose
+  commands:
+    - nosetests
+  imports:
+    - optv
+
+about:
+  home: http://www.openptv.net
+  license:  LGPL v3.0
```

Comment:
Please, add the ``license_file``
Please see:
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=license_file#about